demo/misc.constants

```diff
@@ -1,4 +1,5 @@
 {
+  "cromwell_host": "http://cromwell.example.com:8080",
```

Comment:
Andre said this is no longer used in the demo olives? https://github.com/oicr-gsi/shesmu/pull/1167#discussion_r986171461

Reply:
Great thanks, I hadn't seen that comment